Resources/Textures/Objects/Specific/Research/anomalyscanner.rsi/meta.json

```
@@ -4,11 +4,19 @@
     "x": 32,
     "y": 32
   },
-  "license": "CC0-1.0",
-  "copyright": "Created by EmoGarbage",
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by EmoGarbage, Inhands by TiniestShark based off of /tg station's health analyzer at https://github.com/tgstation/tgstation/commit/c7e5ca401c91cb1351c2f68cda3fe35634c4dba2",
   "states": [
     {
       "name": "icon"
-    }
+    },
+      {
+          "name": "inhand-left",
+          "directions": 4
+      },
+      {
+          "name": "inhand-right",
+          "directions": 4
+      }
```

The indentation seems slightly off?

```suggestion
    {
        "name": "inhand-left",
        "directions": 4
    },
    {
        "name": "inhand-right",
        "directions": 4
    }
```

---

Is there a way to fix that going forward? Every new line always indents and i was getting conflicting info on how to not have that happen.

---

Huh, now that I'm looking at it, it's really weird how we do indentation in json files. The editorconfig should be using 4 spaces per tab, though most of the meta.json files I've seen use 2. I've also seen meta.json files using tabs...

I'd probably just either make all indentations 4 or 2 within the file, or maybe just ask a maintainer what's going on.